iset.mm

```diff
@@ -161647,6 +161780,55 @@ According to Wikipedia ("Integer", 25-May-2019,
     znlidl $p |- ( N e. ZZ -> ( S ` { N } ) e. ( LIdeal ` ZZring ) ) $=
       ( wcel czring crg csn wss cfv clidl zringring snssi zringbas eqid sylancr
       cz rspcl ) BPDEFDBGZPHRAIEJIZDKBPLPESRACMSNQO $.
+
+    znval.u $e |- U = ( ZZring /s ( ZZring ~QG ( S ` { N } ) ) ) $.
+    $( The value of the ` Z/nZ ` structure.  It is defined as the quotient ring
+       ` ZZ / n ZZ ` , with an "artificial" ordering added.  (In other words,
+       ` Z/nZ ` is a _ring_ with an _order_ , but it is not an _ordered ring_ ,
+       which as a term implies that the order is compatible with the ring
+       operations in some way.)  (Contributed by Mario Carneiro, 12-Jun-2015.)
+       (Revised by AV, 13-Jun-2019.) $)
```

Comment:
This comment seems to be just copied and pasted from ~znval, and does not fit here.

Reply:
Not a particularly easy comment to write (as this theorem is basically a stepping stone on the way to https://us.metamath.org/mpeuni/zncrng.html ) but what I came up with was "Making a commutative ring as a quotient of ` ZZ ` and ` n ZZ ` ." (in set.mm and iset.mm) which is loosely based on the https://us.metamath.org/mpeuni/znlidl.html comment.